### ISSUE TITLE
Compiled binary shows nothing inside the app window

### ISSUE THREAD

**Issue by caruccio** [raphamorim/rio#366]
Followed instructions from https://raphamorim.io/rio/docs/install/build-from-source and compiled with no errors.
Running the binary shows a transparent window only.

![Screenshot-2023-12-05-12-51-43](https://github.com/raphamorim/rio/assets/540936/cbe113f6-21c7-4329-bcea-ba36b729380f)

```
$ uname -a 
Linux notebook 5.19.16-200.fc36.x86_64 #1 SMP PREEMPT_DYNAMIC Sun Oct 16 22:50:04 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

$ cat /etc/fedora-release 
Fedora release 36 (Thirty Six)

$ git show --pretty=oneline --name-only
0d49d86044c58257eb90cc39e7ed24f97d5e0380 (HEAD -> main, tag: canary, origin/main, origin/HEAD) corrections on linux for EventLoopExtStartupNotify
frontends/cross-winit/src/router/mod.rs
frontends/cross-winit/src/router/window/mod.rs
```

**Comment by raphamorim** [comment]
hmmm, I think is the latest addition with transparency support. Could you try using 0.0.27 version instead?

**Comment by raphamorim** [comment]
Yea I can confirm as well

![image](https://github.com/raphamorim/rio/assets/3630346/d3ed8378-f585-4466-a9e6-e37c3a9cab56)

apparently some odd behavior similar to what @orhun reported, in some cases it does work and other doesn't.

![image](https://github.com/raphamorim/rio/assets/3630346/4078d2bc-5a50-4c77-9a26-0955f826b0b1)

For now I am disabling default transparency for non-macos in Rio unless the configuration is settled that way.

**Comment by caruccio** [comment]
Tried v0.0.27 with no success. The window is all black.

![Screenshot-2023-12-05-22-02-09](https://github.com/raphamorim/rio/assets/540936/39ede215-a2d7-45cc-8098-de67d75b5ecf)

Also, v0.0.29 still not working, with the only difference that now the background where the window is created becomes tha window background itself. If you move the window, the image "follows" it.

![Screenshot-2023-12-05-22-05-19](https://github.com/raphamorim/rio/assets/540936/27695531-a90a-41bb-b9a6-07f7ed0ecf02)

In both versions the keyboard events are not processed. I've clicked the window than type "date > /tmp/x"  but the file is not created. Looks like shell is running though.

![Screenshot-2023-12-05-22-20-23](https://github.com/raphamorim/rio/assets/540936/5d719325-135d-44db-b1ea-794415f93506)
